src/plugins/kibana_usage_collection/server/collectors/cloud/detector/cloud_detector.ts

```diff
@@ -37,9 +37,9 @@ export class CloudDetector {
   /**
    * Get any cloud details that we have detected.
    */
-  getCloudDetails() {
+  public getCloudDetails = () => {
```

Comment:
I had to look up the subtle binding differences again and cross check the change against how we're class 😅 .